### PR TITLE
Add custom domain to tballiance

### PR DESCRIPTION
**Pull request by emn178**
Add custom domain to tballiance

### CODE REVIEW

File: spaces/tballiance/index.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "network": "42",
   "symbol": "TBA",
   "skin": "tballiance",
+  "domain": "tbavote.joyso.io",
```
Comment (gokaiorg) [inline spaces/tballiance/index.json:7]: To add a custom domain you will need to add your domain in `spaces/domains.json` file also.

File: skins/tballiance.scss [hunk 2]
```diff
@@ -7,4 +7,9 @@
   --border-color: #d1d5da;
   --header-bg: #f7f7f7;
   --block-bg: white;
+
+  .button.button--active {
+    background-color: #21b670;
+    color: white;
+  }
```
Comment (gokaiorg) [inline skins/tballiance.scss:14]: Please use only theme variables. No extra custom styles are allowed.

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: skins/tballiance.scss (hunk 2 above)
Comment (emn178) [reply skins/tballiance.scss:14]: but no variable to change button background

**Comment by gokaiorg** [comment]
We've made a major global update, please pull before commit again your changes.
You can use the documentation to help you.
https://docs.snapshot.page/guides/create-a-space#1-fork-snapshot-spaces-repository-here

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**